### DISCUSSION TITLE
Nginx Proxy Manager implementation

### DISCUSSION THREAD

**Discussion by Koliera** [advplyr/audiobookshelf#238] [Q&A] [answered]
I see the example given to the simple NGINX Reverse Proxy, but did anyone get Nginx Proxy Manager working with this? I was thinking in the sense of just using the "Edit Proxy Host" option for the specific audiobookshelf host?

![image](https://user-images.githubusercontent.com/78056083/145733937-2629fdae-b638-4c51-8d50-961f048953b0.png)

**Comment by ChiefMedicalOfficer** [comment, marked as answer]
> I see the example given to the simple NGINX Reverse Proxy, but did anyone get Nginx Proxy Manager working with this? I was thinking in the sense of just using the "Edit Proxy Host" option for the specific audiobookshelf host?
> 
> ![image](https://user-images.githubusercontent.com/78056083/145733937-2629fdae-b638-4c51-8d50-961f048953b0.png)
> 

I've been using it through Nginx Proxy Manager for a couple of weeks now. Nothing special required.

'Add Proxy Host' > Input details (I'm using a subdomain [audiobookshelf.my.host]) > select all relevant options > Request new SSL cert > Save.

**Comment by Koliera** [comment]
So, it works now for some reason...
I think it was maybe a result of two things - I didn't have webhooks enabled and from what I've read that's a necessity for the app to work. Even after I enabled webhooks, I had to uninstall the app and re-install it. 
Now it seems like as it works as needed.

**Comment by argash** [comment]
I have it working through NPM but did not use custom config. 

![image](https://user-images.githubusercontent.com/293324/148706891-86dc7e67-1863-4f20-b955-fa345a046cd9.png)
![image](https://user-images.githubusercontent.com/293324/148706915-855f7d5e-857e-47d2-ab39-2f556819cf96.png)
